lib/katello/plugin.rb

```diff
@@ -377,12 +377,12 @@ def katello_template_setting_values(name)
         collection: proc { http_proxy_select },
         include_blank: N_("no global default")
 
-      setting 'cdn_ssl_version',
+      setting 'cdn_min_tls_version',
         type: :string,
-        default: nil,
-        full_name: N_('CDN SSL version'),
-        description: N_("SSL version used to communicate with the CDN"),
-        collection: proc { hashify_parameters(Katello::Resources::CDN::SUPPORTED_SSL_VERSIONS) }
+        default: Katello::Resources::CDN::SUPPORTED_TLS_VERSIONS.include?('TLSv1.3') ? 'TLSv1.3' : 'TLSv1.2',
```

Comment:
@ehelms 

I set a default of TLSv1.3, with a fallback to TLSv1.2 in case TLSv1.3 is unavailable for some reason

I then reprovisioned my dev environment to perform a clean test. Indeed, when I loaded the WebUI for the first time, the setting was already populated with the value for TLSv1.3

Following that, I performed basic testing like uploading a subscription manifest and enabling and syncing repositories. All of this was working as expected with TLSv1.3

With the fallback to TLSv1.2 if TLSv1.3 is unavailable, the test suite runs just fine. The TLSv1.3 specific test is skipped since the protocol is unavailable, and `test_http_downloader_default_version` confirmed that TLSv1.2 was the default version.